packages/@lwc/features/src/flags.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+type FeatureFlag = boolean | null;
+
+// Initialize the global configuration object if it isn't initialized already.
+const globalAny: any = global;
```

Comment:
should we do `globalThis`? it will require a polyfill. How is that `global` (from node) transformed by our compile step for browsers? is it `windows`?